bionic/aip/task.py

```diff
@@ -0,0 +1,156 @@
+""" Data model for task running on AI platform
+"""
+import logging
+import os
+import sys
+from dataclasses import dataclass
+from typing import Callable, Optional
+
+import blocks
+import cloudpickle
+import googleapiclient.discovery
+from bionic.aip.future import Future
+from bionic.aip.tune import Tune
```

Comment:
Looks like you still have some hyperparam tuning leftover from your refactor. Can you remove all of the tuning related stuff from this file?